lib/wallet-besu.js

```diff
@@ -0,0 +1,62 @@
+const wallet = require('./wallet.js')
+
+const saveOrionKey = function (orionKey) {
+    localStorage.setItem("orionKey", orionKey)
+}
+
+const loadOrionKey = function () {
+    return localStorage.getItem("orionKey")
+}
+
+const createUserWallet = async function (password, orionKey) {
```

Comment:
Rename the function to just `create`.
We already know that the create function is only for the wallet creation when import this library, so let's simplify the call.